ddtrace/encoding.py

```diff
@@ -73,8 +74,11 @@ def __init__(self):
         self.content_type = 'application/msgpack'
 
     def _encode(self, obj):
-        return msgpack.packb(obj, use_bin_type=True)
-
+        # use_bin_type kwarg only exists since msgpack-python v0.4.0
```

Comment:
It's exactly that one! I think in this way it's cleaner and more "efficient" (in human terms since a `if` is way too fast to call it *efficient*). Good work!